packages/base/src/trace-manager.ts

```diff
@@ -120,10 +122,12 @@ export class TraceManager {
     async closeTrace(traceUUID: string): Promise<void> {
         const traceToClose = this.fOpenTraces.get(traceUUID);
         if (traceToClose) {
-            await this.fTspClient.deleteTrace(traceUUID);
-            const deletedTrace = this.removeTrace(traceUUID);
-            if (deletedTrace) {
-                signalManager().emit(Signals.TRACE_CLOSED, {trace: deletedTrace});
+            const deleteResponse = await this.fTspClient.deleteTrace(traceUUID);
+            if (deleteResponse.isOk()) {
```

Comment:
Same question here, what is the expected behavior when it fails? Maybe this needs to be handled in a different issue about error handling.

Reply:
The important case here is to keep the trace in the manager if CONFLICT 409 is returned, so that it can still be deleted later when the last experiment that contains this trace is closed.

Reply:
Maybe for now we keep the trace for 409, otherwise we remove. Other error handling can be handled as a separate task.